pyproject.toml

```diff
@@ -1,3 +1,22 @@
+[project]
+name = "ECPostcodeLookupPages"
+version = "0.0.0"
+description = "tbd"
```

Comment:
TODO: put something in here :smile:

Reply:
https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/22f65540d3afcfbbb73d982843244f5c6a220683